src/main/java/homework_2/pyramid_printer/PyramidPrinter.java

```diff
@@ -0,0 +1,42 @@
+package homework_2.pyramid_printer;
+
+import java.util.Scanner;
+
+public class PyramidPrinter {
+
+    public void run(){
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter the base of the pyramid, valid value is integer:");
+
+        //reading the value of pyramid's base
+        try {
+            int base = scanner.nextInt();
+            if (base < 0) {
+                throw new Exception();
+            }
+            drawingThePyramid(base);
+        }
+        catch (Exception e){
+            System.out.println("Only 1 non-negative integer is allowed as passed parameter! Please, try again");
+            return;
+        }
+        //adding finally block (28.07.2021)
+        finally {
+            scanner.close();
+        }
+    }
+
+    protected void drawingThePyramid(int base){
+        if (base == 0){
+            System.out.println("");
+            return;
+        }
```

Comment:
Is it needed?

Reply:
You were right, it's no needed. Removed.